src/app/lookups/stat-cards.lookup.ts

```diff
@@ -0,0 +1,70 @@
+import { IMarket } from "@sharedModels/platform-api/responses/markets/market.interface";
+import { StatCardInfo } from "@sharedModels/stat-card-info";
+import { Icons } from "../enums/icons";
+
+export class StatCardLookup {
+  constructor(market?: IMarket) {
+    console.log(market);
+    const info = market;
+    this.setStatCardInfo(info);
+  }
+
+  statCardInfo: StatCardInfo[];
+
+  private setStatCardInfo(info: IMarket): void {
```

Comment:
Can rename this class to `MarketStatCardsLookup` and remove the constructor and `statCardInfo` property. 

Can make this a public static method and rename to `getStatCards` that returns the list of results directly. 

Then this service doesn't need to be injected or instantiated to be used like:

```
this.statCards = MarketStatCardsLookup.getStatCards(this.market);
```

Reply:
@mrtpain Fixed